tests/drivers/watchdog/wdt_error_cases/src/main.c

```diff
@@ -42,7 +42,8 @@
 #define DEFAULT_WINDOW_MIN (0U)
 
 /* Align tests to the specific target: */
-#if defined(CONFIG_SOC_SERIES_NRF54LX) || defined(CONFIG_SOC_NRF9280)
+#if defined(CONFIG_SOC_SERIES_NRF54LX) || defined(CONFIG_SOC_NRF54H20) || \
```

Comment:
The commit is about `nrf54l20` but `nrf54h20` is being added. Is this intentional?

Reply:
nRF54H20 was removed by mistake in this commit:
https://github.com/zephyrproject-rtos/zephyr/commit/07ee796c7d40fe8a7ccc8198aa3ef53c2f371dae#diff-0301c1f4e2a8750e8bd567b1fd052d0d795bfac98cfdfbdbc98504aa81bb8b51L46

Reply:
I can put this to a separate commit if requested.